### PR TITLE
Test on julia 0.7

### LINKED ISSUE #113: v0.11 Failing on Julia v0.7
It seems that the latest tag fails on Julia v0.7, this version does not seem to be tested on Travis.

### PR DESCRIPTION
**Pull request by nicoleepp**
Closes #113

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: julia
 os:
   - linux
 julia:
+  - 0.7
```
Comment (rofinn) [inline .travis.yml]: I'd include a comment to replace `Memento.info(...)` with `info(...)` once we're dropping 0.7.

Comment (ararslan) [reply .travis.yml]: Alternatively, you could do an
```julia
import Memento: info, warn
```
which will shadow the deprecated bindings from Base, with a comment saying to remove the `import` once 0.7 is dropped.

Comment (rofinn) [reply .travis.yml]: Yeah, I like that solution better. I think that's what we used to do anyways.

Comment (nicoleepp) [reply .travis.yml]: Oh shoot, I forgot to test this on 0.7. Seems like it doesn't work properly: https://travis-ci.org/invenia/Memento.jl/jobs/511627026

**Comment by nicoleepp** [comment]
I don't have permissions to merge this :)

**Pull request merged**